### ISSUE TITLE
Test file cannot find elm-explorations Test module

### ISSUE THREAD

**Issue by folkertdev** [elm-tooling/elm-language-server#159]
## Expected Behavior

A file that compiles with no errors using `elm-test make` should display no errors in the editor.

## Current Behavior

The language server immediately shows the error 
```
The TestDeque module has a bad import:
    import Test
I cannot find that module! Is there a typo in the module name? When creating a package, all modules must live in the src/ directory.
```

## Steps to Reproduce (for bugs)

This is hard, @andys8 was unable to reproduce, but I get this error in all test files, which points to a problem with how `elm make` is called. 

The [executed command](https://github.com/elm-tooling/elm-language-server/blob/01dd6fe041ee3530c375941eab19ac0c0f9eaa9e/src/providers/diagnostics/elmMakeDiagnostics.ts#L157) is the correct one using `elm-test make`. 

[Here is a `CocInfo` result](https://gist.github.com/folkertdev/7595d803d33d51f9f3cabb57459d2063)

It very much looks like `elm make` is ran somehow, but it doesn't seem to be ran by the language server. Still the message pops up there. 

A theory is that neovim behaves slightly differently from normal vim. 

1. open file in `tests/` on a package, e.g. `folkertdev/elm-deque`
2. observe the error on line 1

## Your Environment

* Version used: 1.4.1
* Editor used: neovim v0.3.7
* Environment name and version (e.g. node v10.16.0, npm v6.9.0):
* Operating System and version: Linux Mint 19

**Comment by folkertdev** [comment]
I think I've found the cause. 

I ran 
```
:scriptnames
```

and saw that the Ale plugin was still loaded from `~/.local/share/nvim` even though it was not in my `init.vim`. I removed the contents of that folder. I then had to reinstall the plugin manager, but after that everything is now working.

So when upgrading to the language server from a previous setup with neovim, make sure the cached data in  `~/.local/share/nvim` doesn't interfere with your new setup.
